frontend/src/pages/sensitive-content.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <VContentPage>
+    <h1>
+      {{ $t("sensitive.title") }}
+    </h1>
+
+    <i18n path="sensitive.description.content" tag="p">
+      <template #openverse>Openverse</template>
+      <template #wpCoc
+        ><VLink
+          href="https://make.wordpress.org/handbook/community-code-of-conduct/"
+          >{{ $t("sensitive.description.wpCoc") }}</VLink
+        ></template
+      >
+      <template #deiStatement
+        ><VLink
+          href="https://make.wordpress.org/handbook/diversity-equity-and-inclusion-in-wordpress/"
+          >{{ $t("sensitive.description.deiStatement") }}</VLink
+        ></template
+      >
+    </i18n>
+
+    <p>{{ $t("sensitive.sensitivity.what", { openverse: "Openverse" }) }}</p>
+    <p>{{ $t("sensitive.sensitivity.how", { openverse: "Openverse" }) }}</p>
+
+    <!-- TODO
+    <h2 id="on-off">{{ $t("sensitive.onOff.title") }}</h2>
+    <p>{{ $t("sensitive.onOff.description", { openverse: "Openverse" }) }}</p>
+    -->
```

Comment:
Should this still be TODO?

Reply:
I kept this as TODO as this is a TODO in the reference Google doc. Alternatively I can write it directly in the PR and we can iterate on it here?